release.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
this is a work in progress, for now just tucking it here. Will find or create a docker container with hub installed and update readme and possibly tie into a hook.